server/sonar-server/src/main/java/org/sonar/server/project/ws/ProvisionedAction.java

```diff
@@ -82,7 +83,10 @@ public void define(WebService.NewController controller) {
       .addSearchQuery("sonar", "names", "keys")
       .addFieldsParam(POSSIBLE_FIELDS);
 
-    action.setChangelog(new Change("6.4", "The 'uuid' field is deprecated in the response"));
+    action.setChangelog(
+      new Change("6.4", "The 'uuid' field is deprecated in the response"),
+      new Change("6.4", "The fields 'total', 'p' and 'ps' are now named 'total', 'pageIndex' and 'pageSize' in the response " +
+        "and are nested inside an attribute 'paging'"));
```

Comment:
You can use a simpler sentence like "Paging response fields is now in a Paging object"